app/javascript/components/pages/ApartmentShow.js

```diff
@@ -27,41 +29,49 @@ const ApartmentShow = ({apartments}) => {
         />
         <CardBody>
           <CardTitle>
-            {selectedApartment.address}
+
```

Comment:
Can you also remove extra lines with no code please.

Reply:
We removed the empty lines and commented code.  Thanks, Gene